tts/UI.def.lua

```diff
@@ -149,6 +156,20 @@ UI = {}
 ---@field scrollSensitivity nil | number @Default 1.0. A factor/multiplier to augment the scroll speed.
 ---@field itemHeight nil | number
 
+---@shape tts__UIDropdownElement : tts__UIInputElementBase
```

Comment:
Seems like `tts__UIInputElementBase` was removed, yet is still being used. I'm suggesting your reinstate it, however it's worth pointing out nonetheless.